### PR TITLE
feat: on each insert/remove from the shelf, the DB is update accordingly

### LINKED ISSUE #23: Update db info after insert/remove product from the shelf

### PR DESCRIPTION
**Pull request by nicolasfara**
Closes #23

### CODE REVIEW

File: models/product.py [hunk 1]
```diff
@@ -1,63 +1,27 @@
...
-    inPromo = fields.Boolean(required=False, missing=False)
-    promoPrice = fields.Float(required=False, missing=None)
 
-    @post_load
-    def make_user(self, data, **kwargs):
-        #pylint: disable=missing-function-docstring,unused-argument,no-self-use
-        return Product(**data)
-
-class ProductTag:
+class Product(BaseModel):
```
[the remark this answers is not in the document]
Comment (nicolasfara) [reply models/product.py:9]: @sonatype-lift ignore

[the remark this answers is not in the document]
Comment (nicolasfara) [reply models/product.py:9]: @sonatype-lift ignore

[the remark this answers is not in the document]
Comment (nicolasfara) [reply models/product.py:9]: @sonatype-lift ignore

[the remark this answers is not in the document]
Comment (nicolasfara) [reply models/product.py:9]: @sonatype-lift ignore

File: models/product.py [hunk 2]
```diff
@@ -1,63 +1,27 @@
...
+    tag_id: str
+    code: str
+    lot: int
+    name: str
+    price: float
+    expirationDate: str #pylint: disable=invalid-name
+    inPromo: bool #pylint: disable=invalid-name
+    promoPrice: Optional[float] = None #pylint: disable=invalid-name
+
+class ProductTag(BaseModel):
```
[the remark this answers is not in the document]
Comment (nicolasfara) [reply models/product.py:22]: @sonatype-lift ignore

File: models/product.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (nicolasfara) [reply models/product.py:9]: @sonatype-lift ignore

[the remark this answers is not in the document]
Comment (nicolasfara) [reply models/product.py:9]: @sonatype-lift ignore

File: models/product.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (nicolasfara) [reply models/product.py:22]: @sonatype-lift ignore

File: models/product.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (nicolasfara) [reply models/product.py:9]: @sonatype-lift ignore

[the remark this answers is not in the document]
Comment (nicolasfara) [reply models/product.py:9]: @sonatype-lift ignore

File: models/product.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (nicolasfara) [reply models/product.py:22]: @sonatype-lift ignore

**Review by vitlinda — CHANGES_REQUESTED** [review]
Check the doc

**Review by vitlinda — APPROVED** [review]

**Pull request merged**